include/aspect/geometry_model/interface.h

```diff
@@ -144,6 +144,12 @@ namespace aspect
         virtual
         double height_above_reference_surface(const Point<dim> &position) const = 0;
 
+        /**
+         * Converts a Cartesian point into another coordinate system.
```

Comment:
Could you say more specifically what it returns? So for example: `Converts a Cartesian Point into another coordinate system and returns it as a Point.`.
@bangerth Shouldn't this return an array or a NaturalPoint? Might be something for a future pull request.